frontend/src/app/testResults/viewResults/actionMenuModals/TestResultDetailsModal.tsx

```diff
@@ -199,11 +184,11 @@ export const DetachedTestResultDetailsModal = ({
           )}
           <DetailsRow
             label="Pregnant?"
-            value={pregnancy && pregnancyMap[pregnancy]}
+            value={pregnancy && pregnancyMap[pregnancy as keyof Pregnancy]}
```

Comment:
⚠️ Since we are now using the type from the query rather than the loose `Result` type previously defined, this `as keyof` is necessary so that it handles us using the string value as the key